roles/pulibrary.lib-jobs/vars/main.yml

```diff
@@ -19,8 +19,15 @@ rails_app_vars:
     value: "http://www.worldcat.org/webservices/catalog/content"
   - name: WORLDCAT_API_KEY
     value: '{{ vault_wc_key | default("key") }}'
+  - name: APP_SECRET_KEY_BASE
+    value: '{{ secret_key }}'
```

Comment:
this fails (I rebased against master) with an `undefined variable for secret_key`

Reply:
ping @carolyncole sorry this sat for a while.